src/packages/migrate/src/commands/DbCommand.ts

```diff
@@ -20,21 +20,13 @@ ${
   process.platform === 'win32' ? '' : chalk.bold('🏋️  ')
 }Manage your database schema and lifecycle during development.
 
-${chalk.bold.yellow('WARNING')} ${chalk.bold(
-    `Prisma db is currently in Preview (${link('https://pris.ly/d/preview')}).
-There may be bugs and it's not recommended to use it in production environments.`,
-  )}
-${chalk.dim(
-  'When using any of the subcommands below you need to explicitly opt-in via the --preview-feature flag.',
-)}
-
 ${chalk.bold('Flag')}
```

Comment:
This and 2 following lines probably should be in the command itself, not here any more.